commons-rng-simple/src/main/java/org/apache/commons/rng/simple/internal/ProviderBuilder.java

```diff
@@ -48,7 +50,7 @@
 import org.apache.commons.rng.core.source64.XoShiRo256StarStar;
 import org.apache.commons.rng.core.source64.XoShiRo512Plus;
 import org.apache.commons.rng.core.source64.XoShiRo512StarStar;
-
```

Comment:
Remove this empty line and add one after you next import statement.

Reply:
Sir, could you elaborate on that change ?